descwl_coadd/coadd.py

```diff
@@ -49,7 +50,9 @@ class MultiBandCoadds(object):
     coadd_wcs: galsim wcs
         wcs for final cuadd
     coadd_dims: (nx, ny)
-        Currently doing x first rather than row, col
+        Dimensions of the main coadd. Currently doing x first rather than row, col
+    psf_dims: (nx, ny)
+        Dimensions of the PSF coadd. Currently doing x first rather than row, col
```

Comment:
It is used as ny, nx

what made you think it was nx, ny?

Reply:
you wrote this "Currently doing x first rather than row, col" above.

Reply:
fixed